source/io/module_eggShell.js

```diff
@@ -239,6 +246,257 @@
             return result;
         };
 
+        Module.eggShell.typeTopAQSize = function (typePointer) {
```

Comment:
To keep the names consistent can you change  typePointer -> typeRef for all these?

Reply:
I'll do